### DISCUSSION TITLE
H2 Inverter integration

### DISCUSSION THREAD

**Discussion by Robert10xyz** [djansen1987/SAJeSolar#95] [Ideas]
Hi everyone,

For the upcoming releases, I kindly request that all H2 inverter parameters be made available, as it shares the same functionality as the H1. To implement this, it should suffice to modify 'sensor.py' by adding a logical 'or' condition for the H2 inverter wherever self.sensors == "h1" is referenced.

Thank you for your time!

**Comment by djansen1987** [comment]
Hi Robert,

Thank you for your message. Always greate to support mode devices. If you would like support please create a pull request with the code adjustments for the H2. If you are not able to write the code we have to wait for some one who can. It is impossible for me to add support for devices I don't own as their is no api documentation on the api of SAJ as we do no use an official api but their own website api.

If it works with h1 in the config just use that.
